downloader/src/main/kotlin/Downloader.kt

```diff
@@ -80,13 +80,14 @@ object Downloader {
         val downloadDirectory: File,
 
         /**
-         * The source artifact that was downloaded, or null if the download was performed from a [VCS][vcsInfo].
+         * The source artifact that was downloaded, or null if either the download was performed from a [VCS][vcsInfo]
+         * or there was no download because [Package.isMetaDataOnly] was set.
```

Comment:
Maybe better "or there was no download performed at all because [Package.isMetaDataOnly] is true."

Reply:
Done.